build/ubuntu/Dockerfile.builder

```diff
@@ -18,9 +17,9 @@ RUN set -x && \
     cd /wire-server && \
     stack update && \
     echo "allow-different-user: true" >> /root/.stack/config.yaml && \
-    stack build --haddock --dependencies-only haskell-src-exts && \
-    stack build --haddock --no-haddock-hyperlink-source haskell-src-exts && \
-    stack build --pedantic --haddock --test --no-run-tests --bench --no-run-benchmarks --dependencies-only -j${THREADS} && \
+    stack build --dependencies-only haskell-src-exts && \
+    stack build haskell-src-exts && \
+    stack build --pedantic --test --no-run-tests --bench --no-run-benchmarks --dependencies-only -j${THREADS} && \
```

Comment:
why does haddock go away here (sorry, didn't go through the >100 commits individually for context).

Reply:
It is not needed and was making the build slow. I guess this would mean hoogle builds are slightly slow, but I think that is fine.